### PR TITLE
fix(tekton): Pass the updated pipelineConfig, not the stale one

### PR DESCRIPTION
**Pull request by abayer**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description

Without this, we use the original `pipelineConfig`, which may not have its pipelines populated.

#### Special notes for the reviewer(s)

/assign @daveconde 
/assign @hferentschik 

#### Which issue this PR fixes

n/a

### CODE REVIEW

File: pkg/cmd/step/create/step_create_task.go [hunk 1]
```diff
@@ -427,6 +425,7 @@ func (o *StepCreateTaskOptions) GenerateTektonCRDs(packsDir string, projectConfi
 	if projectConfig.NoReleasePrepare {
 		o.NoReleasePrepare = true
 	}
+	pipelineConfig := updatedProjectConfig.PipelineConfig
 	err = o.setVersionOnReleasePipelines(pipelineConfig)
```
Comment (hferentschik) [inline pkg/cmd/step/create/step_create_task.go]: So really, this is where the rub was, right? It passed the original /stale config to `setVersionOnReleasePipelines`.

**Comment by hferentschik** [comment]
Even if I sound like a broken record now, I'd argue it would be worth creating an issue, describing how this bug manifested itself and how one can reproduce it.

File: pkg/cmd/step/create/step_create_task.go (hunk 1 above)
Comment (abayer) [reply pkg/cmd/step/create/step_create_task.go]: Yup, exactly.

**Comment by abayer** [comment]
@hferentschik Nope, you’re 100% right. I’ll do that.

**Pull request merged**